watcher/src/bin/main.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+#![feature(external_doc)]
+#![doc(include = "../../README.md")]
+
+use mc_watcher::{config::WatcherConfig, watcher::Watcher, watcher_db::create_or_open_watcher_db};
+
+use mc_common::logger::{create_app_logger, o};
+use mc_ledger_sync::ReqwestTransactionsFetcher;
+use structopt::StructOpt;
+
+fn main() {
+    mc_common::setup_panic_handler();
+    let (logger, _global_logger_guard) = create_app_logger(o!());
+
+    let config = WatcherConfig::from_args();
+
+    let transactions_fetcher =
+        ReqwestTransactionsFetcher::new(config.tx_source_urls.clone(), logger.clone())
+            .expect("Failed creating ReqwestTransactionsFetcher");
+
+    let watcher_db = create_or_open_watcher_db(
+        config.watcher_db,
+        &transactions_fetcher.source_urls,
+        logger.clone(),
+    )
+    .expect("Could not create or open watcher db");
+    let watcher = Watcher::new(watcher_db, transactions_fetcher, logger);
+    // For now, ignore origin block, as it does not have a signature.
```

Comment:
Why do we need to ignore it? It should be okay for blocks to not have signatures. Does anything break if it starts from 0?

Reply:
Nothing breaks if it starts from 0. The mobilecoind usage of the watcher starts from 0, for example. In the watcher main, I started from 1 since the purpose of the watcher main is just to sync signatures to the watcher_db as a test, and in our test environments, the first block is often very large.